app/main.js

```diff
@@ -163,9 +163,10 @@ const Timer = React.createClass({
       timeRemainingMins: newTimeRemainingMins,
       timeRemainingSecs: newTimeRemainingSecs,
     })
-    this.listenForNorify()
+    this.listenForNotify()
   },
   start: function() {
+    currentTask.paused === 'false'
```

Comment:
This is a variable assignment, so you need to use `=`, not the conditional operator `===`.

Also, `false` is not a string. So, just `false`, with no quotes.